### PR TITLE
relay-release-engineer-playbook.md

### PR DESCRIPTION
**Pull request by EMMLynch**

### CODE REVIEW

File: docs/Relay Release Engineer playbook.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+2. Wednesday-Monday
+
+    a. Use the [Sentry Releases][sentry-releases] report to watch for any new issues
+    
+    b. Co-ordinate, re-tag for [stage fixes][stage-fixes] as needed
+
+
+3. Monday
+    
+    a. Release Readiness Review (led by Easy)
```
Comment (groovecoder) [inline docs/Relay Release Engineer playbook.md:20]: suggestion (blocking): Let's link this to the release readiness review meeting agenda doc. So if that agenda ever changes, we don't have to change it here too.

File: docs/Relay Release Engineer playbook.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+2. Lloan
+3. Luke
+4. Se Yeon
+
+[Release-to-stage]: https://github.com/mozilla/fx-private-relay/blob/main/docs/release_process.md#release-to-stage
+
+
+[sentry-releases]:https://mozilla.sentry.io/releases/
+
+[stage-fixes]: https://github.com/mozilla/fx-private-relay/blob/main/docs/release_process.md#stage-fixes
```
Comment (groovecoder) [inline docs/Relay Release Engineer playbook.md:48]:
```suggestion
[Release-to-stage]: https://github.com/mozilla/fx-private-relay/blob/main/docs/release_process.md#release-to-stage
[sentry-releases]:https://mozilla.sentry.io/releases/
[stage-fixes]: https://github.com/mozilla/fx-private-relay/blob/main/docs/release_process.md#stage-fixes
```

**Review by groovecoder — CHANGES_REQUESTED** [review]

**Comment by lloan** [comment]
Just a minor nit - but it looks like all docs are using kebab-case, with the exception of `api_auth.md` which is using snake case. Should we try to keep it the same? @groovecoder

File: docs/Relay Release Engineer playbook.md (hunk 1 above)
Comment (EMMLynch) [reply docs/Relay Release Engineer playbook.md:20]: @groovecoder, I don't see a release readiness review meeting agenda doc. Can you share the link?

**Comment by groovecoder** [comment]
> Just a minor nit - but it looks like all docs are using kebab-case, with the exception of `api_auth.md` which is using snake case. Should we try to keep it the same? @groovecoder

Oh yeah, that's another good point @lloan. @EMMLynch let's change this filename to `docs/relay-release-engineer-playbook.md`

File: docs/Relay Release Engineer playbook.md [hunk 3]
```diff
@@ -40,9 +40,8 @@
 3. Luke
 4. Se Yeon
 
-[Release-to-stage]: https://github.com/mozilla/fx-private-relay/blob/main/docs/release_process.md#release-to-stage
-
-
+[elease-to-stage]: https://github.com/mozilla/fx-private-relay/blob/main/docs/release_process.md#release-to-stage
```
Comment (groovecoder) [inline docs/Relay Release Engineer playbook.md:43]: change (blocking): 
```suggestion
[release-to-stage]: https://github.com/mozilla/fx-private-relay/blob/main/docs/release_process.md#release-to-stage
```

**Review by groovecoder — CHANGES_REQUESTED** [review]

**Review by groovecoder — APPROVED** [review]

**Pull request merged**